app/controllers/submissions_controller.rb

```diff
@@ -21,12 +21,20 @@ def create
   end
 
   def update
-    @code_review = CodeReview.find(params[:code_review_id])
-    @submission = @code_review.submission_for(current_student)
-    if @submission.update(submission_params)
-      redirect_to course_code_review_path(@code_review.course, @code_review), notice: "Submission updated!"
+    if params['times_submitted']
+      @submission = Submission.find(params[:id])
+      times_submitted = @submission.times_submitted + 1 if params['times_submitted'] == "increment"
+      times_submitted = @submission.times_submitted - 1 if params['times_submitted'] == "decrement"
+      @submission.update_columns(times_submitted: times_submitted)
```

Comment:
Are you sure you need `update_columns` here? It's a dangerous method that skips validations and callbacks, and should typically only be used for performance reasons (to avoid instantiating objects when updating a large number of columns).